content/data-reporting/reports-api/index.mdx

```diff
@@ -200,6 +201,82 @@ group_by=hour" \
 
 <Text>
 
+# Chat Missed
+
+This reports shows you the number of chats with no messages sent by agents (except the first welcome message that always occur in conversations).
+
+## Optional arguments
+
+| Argument | Description |
+|---------|--------------------|
+| `date_from` | `YYYY-MM-DD`, defaults to `the beginning of time` |
+| `date_to` | `YYYY-MM-DD`, defaults to `today` |
+| `timezone` | timezone in the [TZ format](http://en.wikipedia.org/wiki/List_of_tz_database_time_zones) (e.g. America/Phoenix) |
+| `groups[]` | id of the group, not set by default, returns statistics for the specified group |
+| `agents[]` | agent's login, not set by default, return statistics for the specified agent |
```

Comment:
```suggestion
| `agents[]` | agent's login, not set by default, returns statistics for the specified agent |
```

Reply:
fixed in https://github.com/livechat/livechat-public-docs/pull/572/commits/72db3c1bd3011dd7ce571afc3f608468d0a8b343